src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -261,13 +258,26 @@ applyCoinbase v logger dbEnv (Miner mid mks) reward@(ParsedDecimal d) pd parentH
     forkTime = vuln797FixDate v
     fork1_3InEffect = blockTime >= forkTime
     throwCritical = fork1_3InEffect || enfCBFailure
+
     blockTime = blockTimeOf pd
+        -- FIXME it should hold that @blockTime == _blockCreationTime parentHeader@, but
+        -- in some unit test runs that is not the case
+
+    bh = fromIntegral $ _pdBlockHeight pd
+
+    cid = V._chainId parentHeader
+        -- FIXME it should hold that
+        -- @cid == unsafeFromText $ P._chainId $ _pmChainId $ _pdPublicMeta pd@
+        -- but in some unit test runs that is not the case. The chain id in public data is empty.
+
+    chash = Pact.Hash $ encodeToByteString $ _blockHash parentHeader
+        -- FIXME ceck that @ _pdPrevBlockHash pd == encode _blockHash@
+        -- FIXME chash includes the quoted text of the parent header. Is that correct?
 
     tenv = TransactionEnv Transactional dbEnv logger pd noSPVSupport
            Nothing 0.0 rk 0 restrictiveExecutionConfig
     txst = TransactionState mc mempty 0 Nothing (_geGasModel freeGasEnv)
     initState = setModuleCache mc $ initCapabilities [magic_COINBASE]
-    chash = Pact.Hash (sshow $ _blockHash parentHeader)
```

Comment:
`show` on `BlockHash` returns the encoding as JSON string, which includes `"` characters. Is that indented? I guess, even if it unintentional, it's probably too late (and not worth the hassle) to change at this point?

Reply:
No, this is a bug we should fix. Can you file an issue? We need to fork it unfortunately.

Reply:
As @emilypi notes, this is coinbase so this is unused. We can fix